java/ql/src/semmle/code/java/dataflow/internal/FlowSummaryImpl.qll

```diff
@@ -630,7 +630,7 @@ module Private {
       )
     }
 
-    private SummaryComponent interpretComponent(string c) {
+    SummaryComponent interpretComponent(string c) {
```

Comment:
I don't think the minor use of this predicate warrants making it public. The only added reference seems like something that could be written without it.